src/handle-data.ts

```diff
@@ -97,6 +97,7 @@ export function getColumnInfo(layout: TableLayout, colIdx: number, pageColIdx: n
       columnWidth,
       id: `col-${pageColIdx}`,
       label: qFallbackTitle,
+      qLibraryId: isMasterItem ? info.qLibraryId : undefined,
```

Comment:
isnt this the same as
```suggestion
      qLibraryId: info.qLibraryId,
```

Reply:
I think you can remove isMasterItem, since we don't need that bool anymore. And we can always check if qLibraryId exists or not

Reply:
✅ done!